src/main/java/school/lemon/changerequest/java/container/ContainerImpl.java

```diff
@@ -1,38 +1,65 @@
 package school.lemon.changerequest.java.container;
 
-public class ContainerImpl implements Container{
+public class ContainerImpl implements Container {
+
+    private Integer[] container = new Integer[INITIAL_ARRAY_SIZE];
+    private int containerSize = 0;
 
     @Override
     public int size() {
-        // TODO: please implement me
-        return 0;
+        return containerSize;
     }
 
     @Override
     public void clear() {
-        // TODO: please implement me
+        containerSize = 0;
     }
 
     @Override
     public Integer get(int index) {
-        // TODO: please implement me
-        return null;
+        if (index < 0 || index >= containerSize)
+            return null;
+        else
+            return container[index];
     }
 
     @Override
     public void add(int element) {
-        // TODO: please implement me
+        if (containerSize == container.length) {
+            Integer[] tmp = new Integer[containerSize * 2];
+            System.arraycopy(container, 0, tmp, 0, containerSize);
+            container = tmp;
+        }
+        container[containerSize] = element;
+        containerSize++;
     }
 
     @Override
     public boolean add(int element, int index) {
-        // TODO: please implement me
-        return false;
+        if (index < 0 || index > containerSize)
+            return false;
+        if (containerSize == container.length) {
+            Integer[] tmp = new Integer[containerSize * 2];
+            System.arraycopy(container, 0, tmp, 0, index);
```

Comment:
As far as I see - you're duplicating logic from `add` method - you're actually trying to make container bigger, so — please extract this logic into separate method and re-use it where needed.